tidalapi/session.py

```diff
@@ -178,8 +188,22 @@ def __init__(
         for word in token2:
             self.client_id.remove(word)
         self.client_id = "".join(self.client_id)
-        self.client_secret = self.client_id
```

Comment:
I don't think the (non pkce) id and secret should be changed here, we should continue using the one decoded earlier.

Reply:
I have forgot to tidy up some PoC code. Just reverted and pushed it.